integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.5.1
+0.5.2
```

Comment:
We just merged #9163 which also did the same update.

please merge with master and bump this one more time.